lib/get/get-outdated-destination-content.js

```diff
@@ -2,6 +2,7 @@ import Promise from 'bluebird'
 import log from 'npmlog'
 
 const BATCH_CHAR_LIMIT = 1990
+const BATCH_SIZE_LIMIT = 100
```

Comment:
Why limiting this to 100, can't we set the limit based on how many id we have in a batch.
```js
limit: batches.length
```